com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs

```diff
@@ -244,7 +244,7 @@ public partial class HDRenderPipeline
 #if ENABLE_VIRTUALTEXTURES
                     // Note: This pass rely on availability of vtFeedbackBuffer buffer (i.e it need to be write before we read it here)
                     // We don't write it when doing debug mode, FullScreenDebug mode or path tracer. Thus why this pass is call here.
-                    m_VtBufferManager.Resolve(m_RenderGraph, hdCamera, vtFeedbackBuffer);
+                    hdCamera.ResolveVirtualTextureFeedback(m_RenderGraph, vtFeedbackBuffer);
```

Comment:
Question what is happening here with multiple camera? isn't the VTFeedback PR overwrite by each camera result? or there is a process on CPU side in PushFullScreenVTFeedbackDebugTexture which do it, mean that there is some sort of sync point between each camera?

Reply:
This is exactly the change :)
Before this PR that VTBufferManager was part of the HDRenderPipeline but now it is owned by HDCamera. So now each camera has a (CPU, async readback) buffer rather then 1 shared across all cameras. 

This used to be working so I guess this broke recently (maybe with the introduction of RG?)
I added an explicit test for this in HDRP_RuntimeTests (where the other VT tests also live)